src-tauri/src/github/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod release_notes;
```

Comment:
```suggestion
pub mod release_notes;

```

Trailing new line so that any code added in the feature doesn't modify this line.